index.html

```diff
@@ -1,102 +1,115 @@
 ---
 title: Home
+layout: homepage
 redirect_from:
-  - /screenshots/
+- /screenshots/
 ---
 
-{% include header.html %}
-
-<div class="jumbotron">
-  <div class="center">
-    <h1>Meet Minetest.</h1>
-    <h3>A free, open source voxel game engine and game. Fully extendable. You are in control.</h3>
-    <p><a class="btn btn-primary btn-lg" href="#about">Tell me more</a> <a class="btn btn-success btn-lg" href="/downloads/">Download</a></p>
-    <p><small><strong>News:</strong> 0.4.16 released. (June 3<sup>rd</sup> 2017)</small></p>
-  </div>
-</div>
-
-<div class="content">
-  <h1 id="about">About</h1>
-
-  <p>
-    <strong>Minetest is a near-infinite-world block sandbox game and a game engine</strong>, inspired by InfiniMiner, Minecraft, and the like.
-  </p>
-
-  <p>
-    Minetest is available natively for Windows, macOS, GNU/Linux, Android, and FreeBSD. It is Free/Libre and Open Source Software, released under the LGPL 2.1 or later.
-  </p>
-
-  <h1 id="features">Features</h1>
-
-  <div class="row">
-    <div class="col-lg-4">
-      <h4>Near-infinite maps</h4>
+<h1 id="Features">Features</h1>
+<div class="row">
+  <div class="col-lg-6">
+    <h2 id="features">Players</h1>
+    <div>
+      <h4>Choose from Multiple Games</h4>
+      <p>There are many games to choose from. You could survive in a harsh
+        environment, build creatively, or fight other players.
+        Just <a href="/customize#games">download a game</a> or connect to a server.</p>
+    </div>
+    <div>
+      <h4>Near-infinite Maps</h4>
       <p>With a 62000<sup>3</sup> block playing area, there's no running out of space. Yes, the world height is also near-infinite.</p>
     </div>
-    <div class="col-lg-4">
-      <h4>Multiplayer support</h4>
+    <div>
+      <h4>Multiplayer Support</h4>
       <p>Play by yourself, locally with your friends, or online with dozens of players on a server.</p>
     </div>
-    <div class="col-lg-4">
-      <h4>Multiple <em>subgames</em></h4>
-      <p>Want to install dozens of mods at once? You can. Want to play Hunger Games? No problem!</p>
+    <div>
+      <h4>Texture Packs</h4>
+      <p>Not happy with the look of the textures? <a href="/customize/#texture-packs">Change them!</a></p>
+    </div>
+    <div>
+      <h4>Beautiful Map Generators</h4>
+      <p>Choose from many different fast map generators included in the engine,
+         or install one as a mod.</p>
     </div>
   </div>
-  <div class="row">
-    <div class="col-lg-4">
+  <div class="col-lg-6">
+    <h2 id="features">Developers</h1>
+    <div>
+      <h4>Make a Game</h4>
+      <p>Create your own voxel game using our
+        <a href="http://dev.minetest.net/Introduction_to_the_API">Lua API</a>.
+        No need to worry about tricky voxel rendering or networking, only scripts
+        to control game play. Your game will be able to run on Windows,
```

Comment:
Possible improvement: "No need to worry about tricky voxel rendering or networking; just write scripts to control game play."
Though this is still wrong, because... there's far more than this to making a game.